pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md

```diff
@@ -136,4 +187,8 @@ This will give you a lot of information about your containers, including the URL
 
 [ddev]: (https://github.com/drud/ddev)
```

Comment:
None of the link macros seem to be working, as you need to use the correct syntax (see the styleguide [here](https://docs.mautic.org/en/home/contributing#absolute-links)):

```suggestion
[ddev]: <https://github.com/drud/ddev>
```